src/react-vite/03-intro-to-jsx/intro-to-jsx.md

```diff
@@ -55,62 +68,216 @@ Take the below HTML and convert it to JSX:
 
 ### Setup
 
-TODO
+We've created an assets package with images and CSS you'll need to build the application. Install the `place-my-order-assets` package.
```

Comment:
I would add a line after this to make the instruction obvious: https://github.com/bitovi/academy/blob/63c1798f4982759c0c8b238da08475e546a1a72e/src/angular/3-creating-components/creating-components.md#L119